core/templates/pages/skill-editor-page/services/skill-editor-state.service.ts

```diff
@@ -295,6 +295,10 @@ export class SkillEditorStateService {
   setSkillRights(skillRights: SkillRights): void {
     this._setSkillRights(skillRights);
   }
+
+  getPrerequisiteSkillIds(): string[] {
+    return this._skill ? this._skill.getPrerequisiteSkillIds() : [];
```

Comment:
Why add this ternary operator? Moreover, why add this function in the first place?

Reply:
@Jaimin791 This isn't the way to handle this, I'm afraid. You'll need to figure out why this._skill is null or undefined, rather than hiding the error, and fix or explain/document the underlying bug. Otherwise this will end up being hacks built on top of hacks and become hard to maintain.